README.md

```diff
@@ -1,7 +1,18 @@
 # pylibczi
+[![C++ Build & Test](https://github.com/AllenCellModeling/pylibczi/workflows/C%2B%2B%20Build%20%26%20Test/badge.svg)](https://github.com/AllenCellModeling/pylibczi/actions)
+[![Python Build & Test](https://github.com/AllenCellModeling/pylibczi/workflows/Python%20Build%2C%20Test%2C%20%26%20Lint/badge.svg)](https://github.com/AllenCellModeling/pylibczi/actions)
+[![codecov](https://codecov.io/gh/AllenCellModeling/pylibczi/branch/feature/pybind11/graph/badge.svg)](https://codecov.io/gh/AllenCellModeling/pylibczi)
+[![License: GPL v3](https://img.shields.io/badge/License-GPLv3-blue.svg)](https://www.gnu.org/licenses/gpl-3.0)
+
 
 Python module to expose [libCZI](https://github.com/zeiss-microscopy/libCZI) functionality for reading (subset of) Zeiss CZI files and meta-data.
 
+## Usage
```

Comment:
Please add even basic examples here. No need to render images but at the very least, `read` and `read_mosaic` and chunked reading? It just seems odd to basically say: "hey if you want to use this, please go to this jupyter notebook that may or may not render"

Reply:
agree.  `pip install ...`,  `import ...`,  `ndarray=read(...)` would be great.